projects/fsastorefrontlib/src/cms-components/my-account/policy/policy-details/policy-details.component.html

```diff
@@ -77,6 +77,17 @@ <h2 class="heading-headline p-3 mt-3 mb-5 text-center">
             {{ policy.endDate | cxDate: 'dd MMM yyyy' }}
           </div>
         </li>
+        <li *ngIf="policy.policyStatus" class="accordion-list-item row">
+          <div class="col-6 col-sm-5 item-label">
+            {{ 'fscommon.status' | cxTranslate }}
+          </div>
+          <div class="col-6 col-sm-5 value notice">
+            {{
```

Comment:
Seems like way too much logic just to have capitalized word. I would try just ```policy.policyStatus.toLowerCase()``` and then `text-capitalize` class on div.
But even with that I think we should avoid having this logic just for setting word capitalization. But for that best to consult with @anajanjusevic.

Reply:
I think this formatting should come from backend. It's Enum value, so it should be localized and everything in pooplator. Here we should just display it.